libpod/image/pull.go

```diff
@@ -259,7 +259,7 @@ func (i *Image) pullImage(ctx context.Context, writer io.Writer, authfile, signa
 		if !hasRegistryInName && len(searchRegistries) == 0 {
 			return nil, errors.Errorf("image name provided is a short name and no search registries are defined in %s.", registryPath)
 		}
-		return nil, errors.Errorf("unable to find image in the registries defined in %q", registryPath)
+		return nil, errors.Errorf("unable to find image or you do not have pull access")
```

Comment:
Could you leave in the registryPath?  Or even better add the searchRegistries and the registryPath.

Unable to find image in REGISTRIESLIST defined in REGISTRYPATH, or you do not have pull access from these registries.

Reply:
I like @rhatdan proposal.

Users coming from Docker usually have some (minor) troubles wrapping their heads around how `registries.conf` works (and most don't even know about its existence). Pointing there directly when pulling fails will draw attention to the right place.

Reply:
yes, and then if it still fails, hopefully the "or you don't have pull access" part will make them think ``podman logout ... && podman login ...``

Reply:
pushed with my compromised solution.